src/backend/EdFi.DataManagementService.Backend.Installer/README.md

```diff
@@ -0,0 +1,3 @@
+Example for running DB install from Linux, from the `src` directory:
+
+`dotnet run -r linux-x64 --project backend/EdFi.DataManagementService.Backend.Installer/EdFi.DataManagementService.Backend.Installer.csproj -- postgresql 'host=localhost;port=5432;username=postgres;password=abcdefgh1!;database=EdFi.DataManagementService'`
```

Comment:
Suggest using full code fencing here with triple backticks, and splitting into separate lines for better readability with the `\` line continuation character.